frontend/src/components/TutorPage.js

```diff
@@ -81,15 +82,11 @@ export default function TutorPage() {
             },
           }}
         >
-          <Avatar
-            sx={{
-              mt: { md: "5px" },
-              bgcolor: blue[500],
-              width: { xs: 50, md: 100 },
-              height: { xs: 50, md: 100 },
-            }}
-            alt={data.tutor.firstname + " " + data.tutor.lastname}
```

Comment:
We should keep the alt text for accessibility. It should probably display `<Username>'s Avatar` or something instead of their name though.